libbeat/autodiscover/providers/kubernetes/kubernetes.go

```diff
@@ -354,17 +354,3 @@ func (p *leaderElectionManager) startLeaderElector(ctx context.Context, lec lead
 	p.logger.Debugf("Starting Leader Elector")
 	go le.Run(ctx)
 }
-
-func ShouldPut(event mapstr.M, field string, value interface{}, logger *logp.Logger) {
```

Comment:
are those changes, like renaming `ShouldPut` -> `util.ShouldPut` suppose to be in this PR?